internal/engine/buildcontrol/image_build_and_deployer_test.go

```diff
@@ -869,6 +870,8 @@ func TestDockerBuildStatus(t *testing.T) {
 	_, err = f.BuildAndDeploy(BuildTargets(manifest), store.BuildStateSet{})
 	require.NoError(t, err)
 
+	f.ibd.dr.Reconcile(f.ctx, ctrl.Request{NamespacedName: nn})
```

Comment:
Would it make sense to move this into `ibdFixture::BuildAndDeploy`?

Reply:
done